piet-common/tests/image.rs

```diff
@@ -66,3 +66,19 @@ fn empty_image_area_should_not_panic() {
         Ok(())
     })
 }
+
+#[test]
+fn image_size() {
+    let image = ImageBuf::from_raw(&[0, 0, 0, 0][..], ImageFormat::RgbaSeparate, 1, 1);
+    with_context(|ctx| {
+        let image = image.to_image(ctx);
+        if image.size() != Size::new(1., 1.) {
+            return Err(format!(
+                "expected {:?}, found {:?}",
+                Size::new(1., 1.),
+                image.size()
+            ));
+        }
+        Ok(())
+    })
+}
```

Comment:
do we also want to explicitly test a zero-sized image? I'm not sure what our expectations are there, now...

Reply:
Yeah, good idea.